lighthouse-core/gather/gatherers/fonts.js

```diff
@@ -146,7 +146,10 @@ class Fonts extends Gatherer {
     ).then(([loadedFonts, fontFaces]) => {
       return loadedFonts.map(fontFace => {
         const fontFaceItem = this._findSameFontFamily(fontFace, fontFaces);
-        fontFace.src = fontFaceItem.src || [];
+        fontFace.src = [];
```

Comment:
seems like we lost the case where `.src` is not an array but not sure if that's possible?

how abouts `fontFace.src = (fontFaceItem && fontFaceItem.src) || []`?

Reply:
your wish is my command!